### ISSUE TITLE
2:1 AV block

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7591]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of second-degree atrioventrivucarl block in which every other P wave is not conducted through the AV node such that only every other P wave is followed by a QRS complex. 
Also revise definitions of related terms

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 616247   PMID:23388215 1/1
 616249  PMID:23388215 1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
603830    PMID: 15184283  1/1

**Comment by pnrobinson** [comment]
added
